daisy_workflows/image_build/install_package/cos/package_replacement/disable_readonly_rootfs.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#
+# Disables vboot and unlocks rootfs.
+
+set -o errexit
+set -o pipefail
+set -x
+
+source /usr/share/vboot/bin/common_minimal.sh
+
+# This procedure may or may not trigger an immediate reboot. Cos-customizer is
+# designed to re-run (and only from) the script that triggered the reboot, so
+# it is important that this procedure handles re-entry gracefully.
+disable_vboot() {
```

Comment:
so we are changing/deriving the running system and rebooting it?

Reply:
Yes, we disable read-only, perform replacement, re-enable read-only and create a resulting image.